plugins/woocommerce-admin/client/shipping/shipping-tour.tsx

```diff
@@ -0,0 +1,114 @@
+/**
+ * External dependencies
+ */
+import { TourKit, TourKitTypes } from '@woocommerce/components';
+import { __ } from '@wordpress/i18n';
+import { useDispatch, useSelect } from '@wordpress/data';
+import { OPTIONS_STORE_NAME } from '@woocommerce/data';
+
+const REVIEWED_DEFAULTS_OPTION =
+	'woocommerce_admin_reviewed_default_shipping_zones';
+
+const CREATED_DEFAULTS_OPTION =
+	'woocommerce_admin_created_default_shipping_zones';
+
+const useShowShippingTour = () => {
+	const {
+		hasCreatedDefaultShippingZones,
+		hasReviewedDefaultShippingOptions,
+		isLoading,
+	} = useSelect( ( select ) => {
+		const { hasFinishedResolution, getOption } =
+			select( OPTIONS_STORE_NAME );
+
+		return {
+			isLoading:
+				! hasFinishedResolution( 'getOption', [
+					CREATED_DEFAULTS_OPTION,
+				] ) &&
+				! hasFinishedResolution( 'getOption', [
+					REVIEWED_DEFAULTS_OPTION,
+				] ),
+			hasCreatedDefaultShippingZones:
+				getOption( CREATED_DEFAULTS_OPTION ) === 'yes',
+			hasReviewedDefaultShippingOptions:
+				getOption( REVIEWED_DEFAULTS_OPTION ) === 'yes',
+		};
+	} );
+
+	return {
+		isLoading,
+		show:
+			! isLoading &&
+			hasCreatedDefaultShippingZones &&
+			! hasReviewedDefaultShippingOptions,
+	};
+};
+
+export const ShippingTour = () => {
+	const { updateOptions } = useDispatch( OPTIONS_STORE_NAME );
+	const { show: showTour } = useShowShippingTour();
+
+	const tourConfig: TourKitTypes.WooConfig = {
+		placement: 'auto',
+		steps: [
```

Comment:
> There should be 3 steps in the tour, covering shipping zones, shipping methods and recommended shipping options

@rjchow I was not able to see the 3 steps. I think there are only 2 steps in this PR?

Reply:
Silly me I forgot to push the latest!! Fixed